lib/mongodb.js

```diff
@@ -191,8 +196,38 @@ MongoDB.prototype.fromDatabase = function(model, data) {
         // Convert the Binary into String
         data[p] = data[p].toString();
       }
+    } else if (data[p] && prop && prop.type && prop.type.name === 'GeoPoint' &&
+      this.settings.enableGeoIndexing === true) {
```

Comment:
Please indent by two levels, otherwise it's difficult to distinguish what's part of the `if` condition and where the branch body begins.

```js
    } else if (data[p] && prop && prop.type && prop.type.name === 'GeoPoint' &&
        this.settings.enableGeoIndexing === true) {
      data[p] = // ...
    }